samtranslator/swagger/swagger.py

```diff
@@ -695,6 +700,10 @@ def add_auth_to_method(self, path: str, method_name: str, auth: Dict[str, Any],
             self._set_method_authorizer(path, method_name, method_authorizer, authorizers, method_scopes)  # type: ignore[no-untyped-call]
 
         method_apikey_required = auth and auth.get("ApiKeyRequired")
+
+        if not auth.get("AddApiKeyRequiredToCorsPreflight") and method_name == "options":
```

Comment:
Is the type of `AddApiKeyRequiredToCorsPreflight` checked anywhere?

Currently if it's `"False"` (a string), `not "False"` will evaluate `False`.

Reply:
I don't think it is checked anywhere. Wouldn't "False" (a string) be an incorrect input or should that still be an acceptable input?

Reply:
I don't know. If you're unsure, add tests for that case.